app/services/validators/validation-error-messages.js

```diff
@@ -1,11 +1,13 @@
 module.exports = {
+  getIsOlderThanMaxHistory: function (displayName, options) { return `Date range must be within the past ${options.years} years` },
   getIsRequiredMessage: function (displayName) { return `${displayName} is required` },
   getIsAlphaMessage: function (displayName) { return `${displayName} must only contain letters` },
   getIsNumericMessage: function (displayName) { return `${displayName} must only contain numbers` },
   getIsRangeMessage: function (displayName, options) { return `${displayName} must be between ${options.min} and ${options.max}` },
   getIsIntegerMessage: function (displayName, options) { return `${displayName} must be a whole number between ${options.min} and ${options.max}` },
   getIsFloatMessage: function (displayName, options) { return `${displayName} must be a number between ${options.min} and ${options.max}` },
   getInvalidDateFormatMessage: function (displayName) { return `${displayName} was invalid` },
+  getPastDateMessage: function (displayName) { return `${displayName} must be in the past` },
   getFutureDateMessage: function (displayName) { return `${displayName} must be in the future` },
   getIsDateLaterThanMessage: function (displayName) { return `${displayName} must be after the start date` },
```

Comment:
If I put in a really old "Show capacity from", e.g. 01 01 1000 the message is "Show capacity from must be after the start date"

If I put in two current dates that are the same, e.g. both 01 09 2017 the message is "Show capacity to must be after the start date"

This is slightly confusing as to what "start date" is. Is it the "Show capacity from" date, or the date at which we started having data?